R/client.R

```diff
@@ -367,9 +437,6 @@ SapiClient <- setRefClass(
             }}
             \\subsection{Return Value}{URL to ping for table creation status check.}"
             opts <- .self$prepareOptions(options)
-            if (!("federationToken" %in% names(opts))) {
-                opts$federationToken = 1
-            }
```

Comment:
why do we want to remove this default?

Reply:
because there no such parameter any more https://keboola.docs.apiary.io/#reference/tables/unload-data-asynchronously/asynchronous-export - i.e. it is no longer valid for tables, for files it is still used here https://github.com/keboola/sapi-r-client/blob/1078d87de73c6a436499de69239fe29e628dd754/R/client.R#L187 and here https://github.com/keboola/sapi-r-client/blob/1078d87de73c6a436499de69239fe29e628dd754/R/client.R#L328

Reply:
aha 👍